services/wallet/reader.go

```diff
@@ -72,10 +72,11 @@ type TokenMarketValues struct {
 }
 
 type ChainBalance struct {
-	Balance  *big.Float     `json:"balance"`
-	Address  common.Address `json:"address"`
-	ChainID  uint64         `json:"chainId"`
-	HasError bool           `json:"hasError"`
+	RawBalance string         `json:"rawBalance"`
```

Comment:
Just curious why raw balance is string?

Reply:
initially I wanted to use `*big.Int` but it turns out that then serialization doesn't work properly - looks like nim tries to deserialize to a plain number which can be insufficient to handle amount.

Reply:
> initially I wanted to use `*big.Int` but it turns out that then serialization doesn't work properly - looks like nim tries to deserialize to a plain number which can be insufficient to handle amount.

I think this is the main reason why we usually expose `big.Int` as [*hexutil.Big](https://github.com/status-im/status-go/blob/2f539d3bd23694f1d0db7fdbf1ffde81f42e8d7d/vendor/github.com/ethereum/go-ethereum/common/hexutil/json.go) usually in the status-go API.